arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/README.md

```diff
@@ -0,0 +1,30 @@
+# Graphics overlay (dictionary renderer) 3D
+
+Display MIL-STD-2525D military symbology in 3D.
+
+The dictionary renderer creates these graphics using the mil2525d style file plus an XML file with key-value attributes for each graphic.
+
+![](image1.png)
+
+## How it works
+
+To apply an `AGSDictionaryRenderer` and display mil2525d graphics in 3D:
+
+1. Create an `AGSDictionarySymbolStyle` object with the "mil2525d" specification type.
+2. Create an `AGSDictionaryRenderer` object with the dictionary symbol style.
+3. Create an instance of `AGSGraphicsOverlay`.
+4. Set the dictionary renderer as the renderer of the graphics overlay.
+5. Parse an XML file conforming to the specification. It should have key-value pairs to use as attributes for each graphic:
```

Comment:
I don't know much about this api - but it seems like we should have something in the api that can parse these and create graphics for the user. Are we sure that doesn't exist in our api?

Reply:
I'm fairly certain, but I'll double check. My understanding is that the XML schema is dependent on the specification, so it wouldn't be possible to create a single parser that works for all specifications.